changelog

```diff
@@ -1,3 +1,142 @@
+      * Added defense fleets to Wanderer worlds. (@TheGiraffe3)
+      * Made systems in the Umbral Reach shrouded and added a flavor mission. (@Saugia)
+      * Added some new news messages. (@petervdmeer)
+      * Adjusted some fleets and minables in Gegno space. (@Saugia)
+      * Added log entries for the Incipias. (@roadrunner56)
+      * Added the battle of Zydee and a new planet in Gegno space, along with various tweaks. (@Saugia)
+      * Added more hostile disabled pirate hails. (@blue-four-seven)
+      * Added Quarg poems about the Incipias and Successors. (@Arachi-Lover)
+      * Added more friendly Hai hails. (@blue-four-seven)
+      * Added unique conversations for Geminus and Martini during the bombings. (@mOctave)
+      * Added spaceport news and culture conversations to CCOR worlds. (@Ebreus, @bene-dictator)
+      * Added some new friendly civilian hails. (@blue-four-seven)
+    * Graphics:
+      * Added a variety of new Landscape Images. (@tekker2234)
+      * Reverted changes to the "land/water4" image. (@ziproot)
+    * Balance:
+      * Adjusted the integrated stats of the Penguin. (@ziproot)
+        * "heat dissipation": 0.82 -> 0.88
+        * Added "energy generation": 0.6
+        * Added "heat generation": 1
+        * "thrust": 4.2 -> 6.3
+        * Added "thrusting energy": 0.6
+        * Added "thrusting heat": 0.55
+        * "turn": 80 -> 120
+        * Added "turning energy": 0.22
+        * Added "turning heat": 0.24
+        * "reverse thrust": 4.2 -> 6.3
+        * Added "reverse thrusting energy": 0.55
+        * Added "reverse thrusting heat": 0.55
+      * Gave the Fetri'sei logical stats for its category and description, replacing stats copied from the Maeri'het. (@Quantumshark)
+      * Made the Dragonflame penetrate instead of having a blast radius. (@AvianGeneticist)
+      * Reduced the power of the Merganser, the Peregrine and the Bellows-class Afterburner. (@Azure_)
+    * Other:
+      * Adjusted wording in "Pookie, Part 2" to more accurately reflect the behavior of dogs. (@aradapilot)
+      * Doubled the average interval for "person" ship spawns. (@Saugia)
+      * Added formation patterns for person ships. (@petervdmeer)
+      * Edited the description of Ghneoe. (@TheGiraffe3)
+      * The "Wanderer Freight" fleet now has the "appeasing" personality. (@warp-core)
+  * Game mechanics:
+    * New mechanics:
+      * Dialog text, and planet and spaceport descriptions can now have "to display" condition sets. (@UnorderedSigh)
+      * References to phrases and other substitutions will now be expanded in custom substitutions, and NPC and gift ship names. (@UnorderedSigh)
+      * Added formation flying with simple patterns. (@petervdmeer)
+      * Outfits can now specify custom scan sounds. (@TomGoodIdea)
+      * Added "<model>" text replacement for model name of the flagship. (@TomGoodIdea)
+      * Made ships snap to formation. (@tibetiroka)
+      * Complex fleet formations. (@petervdmeer)
```

Comment:
Feel free to combine this with line 90. (Just change that line to "Added fleet formations", then it covers the code and the simple and the complex patterns all in 3 words.)